src/components/MultiHopTrade/components/FiatRamps/hooks/useGetRampQuotes.tsx

```diff
@@ -60,31 +53,35 @@ export const useGetRampQuotes = ({
     queries:
       supportedRamps?.map(fiatRamp => ({
         queryKey: [...queryKey, fiatRamp.id],
-        queryFn: () => {
-          switch (fiatRamp.id) {
-            case 'OnRamper': {
-              if (!onramperCurrencies) throw new Error('Onramper currencies not found')
-              const crypto = findOnramperTokenIdByAssetId(assetId, onramperCurrencies)
-              if (!crypto) throw new Error('Asset not found')
+        queryFn: async () => {
+          const result = await (async () => {
```

Comment:
That looks off - all we do with result IIAFE is get this out:

<img width="444" height="77" alt="Image" src="https://github.com/user-attachments/assets/8f07125f-126d-4868-8549-b8a6fd9a0853" />

Why not have consistent `RampQuote | null` and remove this extra useless IIAFE altogether?

something like that

```typescript`
queryFn: async () => {
  switch (fiatRamp.id) {
    case "OnRamper": {
      // logic goes brr here
      return (await fiatRamp.getQuotes?.({ /* ... */ })) || null
    }
    default: {
      // logic goes brr here
      return (await fiatRamp.getQuotes?.({ /* ... */ })) || null
    }
  }
}